sound/soc/codecs/rt711.c

```diff
@@ -1206,8 +1213,25 @@ int rt711_init(struct device *dev, struct regmap *sdw_regmap,
 				&soc_codec_dev_rt711,
 				rt711_dai,
 				ARRAY_SIZE(rt711_dai));
+	if (ret < 0)
+		return ret;
+
+	/* set autosuspend parameters */
+	pm_runtime_set_autosuspend_delay(dev, 3000);
```

Comment:
in SOF driver we set it to SND_SOF_SUSPEND_DELAY_MS = 2000. I don't know why it is 3000 ?  And 3000 is used for all sdw codec can we use a macro definition to replace it ?